android/engine/src/main/java/org/smartregister/fhircore/engine/ui/questionnaire/QuestionnaireViewModel.kt

```diff
@@ -143,6 +145,13 @@ constructor(
 
               if (resource is Patient) resource.managingOrganization = organizationRef
               else if (resource is Group) resource.managingEntity = organizationRef
+              // TODO: calculate birthDate from AgeInput as calculation isn't supported by sdk
+              // https://github.com/google/android-fhir/issues/803
```

Comment:
according to this issue comment, https://github.com/google/android-fhir/issues/803#issuecomment-935834791, all the pre-req issues for this functionality are now closed.

is this now supported?

if so, please remove this and use a calculation.

if not, please add a comment on the issue specify what is remaining so there's a path to closing it and using it here to remove this custom code

Reply:
**calculated-expression and variable not working - FHIR Reference**
This point is tracked by issue and is not supported
https://github.com/google/android-fhir/wiki/Structured-Data-Capture-Library-Developer's-Guide here calculation is not mentioned in supported list

Reply:
Can you please add a comment to https://github.com/google/android-fhir/issues/803 noting that and describing what remains to be implemented?

All the tracking issues in that issue are closed, so it implies that everything's done for calculations to be supported, but like you've said that's not correct, so we need to make sure that is clear and what work remains to be done for this to be supported.

Reply:
It has been discussed with @maimoonak **calculated-expression and variable ** is yet not supported so as per mutual understanding I have added that //todo comment, and implemented the ticket requirements as per suggestion provided by Maimoona. She has already posted comment to google/android-fhir#803.

Reply:
Looks good, thanks @AbdulWahabMemon